modules/python/src2/cv2.cpp

```diff
@@ -1772,7 +1777,15 @@ bool pyopencv_to(PyObject* obj, Rect& r, const ArgInfo& info)
     else
     {
         std::vector<int> value(4);
-        pyopencvVecConverter<int>::to(obj, value, info);
+        if (!pyopencvVecConverter<int>::to(obj, value, info))
```

Comment:
I think this part wont be needed after #19344 merge, because it handles all Python sequence in a common way.

Reply:
Yes, I'm aware of this patch and agree that implementation will be replaced.